sycl/test-e2e/BFloat16/bfloat16_limits.cpp

```diff
@@ -0,0 +1,103 @@
+/// Checks a numeric_limits specialization of bfloat16.
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
```

Comment:
Looks like this belongs to `sycl/test`, not `sycl/test-e2e`

Reply:
Or even `sycl/unittests`